ui/tests/integration/components/sync/secrets/page/overview-test.js

```diff
@@ -25,18 +25,20 @@ module('Integration | Component | sync | Page::Overview', function (hooks) {
 
   hooks.beforeEach(async function () {
     this.version = this.owner.lookup('service:version');
+    this.persona = this.owner.lookup('service:persona');
     this.version.type = 'enterprise';
     this.version.features = ['Secrets Sync'];
+    this.version.activatedFeatures = ['secrets-sync'];
     syncScenario(this.server);
     syncHandlers(this.server);
 
     const store = this.owner.lookup('service:store');
     this.destinations = await store.query('sync/destination', {});
-    this.isActivated = true;
 
-    this.renderComponent = () => {
+    this.renderComponent = (persona) => {
+      this.persona = persona || this.persona.secretsSyncPersona;
       return render(
-        hbs`<Secrets::Page::Overview @destinations={{this.destinations}} @totalVaultSecrets={{7}} @isActivated={{this.isActivated}} />`,
+        hbs`<Secrets::Page::Overview @destinations={{this.destinations}} @totalVaultSecrets={{7}} @secretsSyncPersona={{this.persona}} />`,
```

Comment:
Same here -- the tests can be updated to reflect the new behavior, which is something like "it should show upsell CTA if persona is <PERSONA>" and then you probably won't need to stub any of the services